tests/api.c

```diff
@@ -33606,12 +33607,16 @@ static int test_wolfSSL_IMPLEMENT_ASN1_FUNCTIONS(void)
     XFREE(der, NULL, DYNAMIC_TYPE_ASN1);
     DPP_BOOTSTRAPPING_KEY_free(bootstrap);
 
+    /* Test integer */
+    AssertNotNull(test_asn1 = TEST_ASN1_new());
+    der = NULL;
+    AssertIntEQ(i2d_TEST_ASN1(test_asn1, &der), 4);
```

Comment:
This test case is failing with: ./configure CC="clang -fsanitize=address,undefined -g" --enable-all --enable-debug

There is not an integer value put into test_asn1 which results in an error when wolfSSL_i2d_ASN1_INTEGER is called which messes up the length.

Reply:
Done.